ape_alchemy/provider.py

```diff
@@ -26,6 +27,12 @@ class Alchemy(Web3Provider, UpstreamProvider):
 
     network_uris: Dict[tuple, str] = {}
 
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.block_page_size = 5000
+        self.concurrency = 1
+        print(f"setting block_page_size to {self.block_page_size} and concurrency to {self.concurrency}")
```

Comment:
- [ ] Replace `print()` statements with the logger.

```python
from ape.logging import logger

logger.debug()
# or
logger.info()
# etc.
```

Reply:
removed